deployment/helm/skaha/sample-local-values.yaml

```diff
@@ -91,3 +96,49 @@ storage:
       # storageClassName: ""
       labels:
         storage: local-storage  # Match this label up with whatever was installed in the base install, or the desired PVC
+
+# Kueue setups
+kueue:
```

Comment:
Do the values here represent good defaults?

Reply:
Yes these are good defaults, however resource quotas can be changed according to availability of the resource in the cluster.